packages/cadl-ranch-specs/http/authentication/multiple/main.cadl

```diff
@@ -0,0 +1,33 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+
+@scenarioService("/authentication/multiple")
```

Comment:
nit: do we want to specify this scenario is a choice of multiple auth vs having multiple on top of each other?

Maybe `multiple/options` or just options?